tests/input_data/db/test_demographics.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+import pandas as pd
+
+from cascade.input_data.db.demographics import age_ranges_to_groups
+
+
+def test_age_ranges_to_groups(mocker):
+    fake_get_age_groups = mocker.patch("cascade.input_data.db.demographics.get_age_groups")
+    fake_get_age_groups.return_value = pd.DataFrame(
+        {"age_group_id": [1, 2, 3, 4], "age_group_years_start": [0, 1, 25, 90], "age_group_years_end": [1, 2, 30, 100]}
+    )
+
+    df = pd.DataFrame({"age_lower": [0, 1, 90], "age_upper": [1, 2, 100], "integrand": ["remission"] * 3})
```

Comment:
Good. This tests the inverse problem. That part always bothered me. It tests the age part.